BlotoutAnalytics/Foundation/BOFUtilities/BOReachability.m

```diff
@@ -189,6 +188,8 @@ - (BOOL)startNotifier
 {
 	BOOL returnValue = NO;
 	SCNetworkReachabilityContext context = {0, (__bridge void *)(self), NULL, NULL, NULL};
+    
+    if(reachabilityRef)
```

Comment:
@nitinblotout can you please add brackets around code as right now it looks like a bug

Reply:
Done.